sdk/identity/Azure.Identity/src/Constants.cs

```diff
@@ -41,5 +41,8 @@ internal class Constants
         public static readonly KeyValuePair<string, string> DefaultMsaltokenCacheKeyringAttribute2 = new KeyValuePair<string, string>("Microsoft.Developer.IdentityService", "1.0.0.0");
 
         public const string DefaultMsalTokenCacheName = "msal.cache";
+
+        public const string ManagedIdentityClientId = "client_id";
```

Comment:
FYI, this would be incorrect for App Service 2017-09-01, which requires "clientid".

Reply:
Yes, it's only used where it is correct.